libs/blocks/global-navigation/blocks/appLauncher/appLauncher.js

```diff
@@ -46,13 +46,13 @@ function decorateAppsMenu(profileEl, appsDom, toggle) {
   return appsNavItem;
 }
 
-export default async function getApps(profileEl, appLauncherBlock, toggle) {
+async function appLauncher(profileEl, appLauncherBlock, toggle) {
   const gnav = profileEl.closest('nav.gnav');
   gnav.classList.add('has-apps');
```

Comment:
As a general observation, we should always check for the element existence before doing any changes. We want to avoid errors like _cannot read property of undefined_